app/views/matches/index.html.slim

```diff
@@ -23,17 +25,17 @@
           - if match['winner']=='b'
             span.blue-text
               = match['blue_goal']
-            | -
+            | &nbsp-&nbsp
```

Comment:
Try not to manually escape HTML; otherwise we wouldn't be able to read what you are trying to do. 

You can google for rails helpers to escape strings; I believe there are a few.